cmd/cmd.go

```diff
@@ -491,7 +491,7 @@ func resolveExitCode(err error) {
 		os.Exit(exitcode.TransferExceeded)
 	case fserrors.ShouldRetry(err):
 		os.Exit(exitcode.RetryError)
-	case fserrors.IsNoRetryError(err):
+	case fserrors.IsNoRetryError(err), fserrors.IsNoLowLevelRetryError(err):
```

Comment:
Shouldn't that say 

    fserrors.IsNoRetryError(err) || fserrors.IsNoLowLevelRetryError(err)

Or is this some case syntax I'm not aware of...

Turns out it is syntax I'm not aware of: https://go.dev/play/p/i6tDdKufO3L !